### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Vasyl-Poremchuk**

### CODE REVIEW

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Comment by Vasyl-Poremchuk** [comment]
image

**Review by wQuelS — APPROVED** [review]

File: taxi/views.py [hunk 1]
```diff
@@ -87,3 +88,29 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
```
[the remark this answers is not in the document]
Comment (Vasyl-Poremchuk) [reply taxi/views.py:90]: Thanks, I'll change it)

File: taxi/views.py [hunk 2]
```diff
@@ -86,4 +87,30 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
-    queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+    queryset = Driver.objects.prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
```
Comment (Nattalli) [inline taxi/views.py:95]: what about `success_url`?

**Review by Nattalli — CHANGES_REQUESTED** [review]

**Review by Y-Havryliv — APPROVED** [review]
GJ!

**Pull request closed without merging**